testing/tests/redirects.test.js

```diff
@@ -0,0 +1,1034 @@
+    "/docs/Web/API/Canvas_API/Tutorial/Transformations#A_save_and_restore_canvas_state_example"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/5_2_canvas_translate.html",
+    "/docs/Web/API/CanvasRenderingContext2D.translate"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/5_3_canvas_rotate.html",
+    "/docs/Web/API/CanvasRenderingContext2D.rotate"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/5_4_canvas_scale.html",
+    "/docs/Web/API/CanvasRenderingContext2D.scale"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/6_1_canvas_composite.html",
+    "/docs/Web/API/CanvasRenderingContext2D.globalCompositeOperation"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/6_2_canvas_clipping.html",
+    "/docs/Web/API/Canvas_API/Tutorial/Compositing#Clipping_paths"
+  ),
+  url_test(
+    "/samples/canvas-tutorial/globalCompositeOperation.html",
+    "/docs/Web/API/CanvasRenderingContext2D.globalCompositeOperation"
+  ),
+  url_test(
+    "/samples/domref/mozGetAsFile.html",
+    "/docs/Web/API/HTMLCanvasElement.mozGetAsFile"
+  ),
+  url_test("/Firefox_OS/Security", "/docs/Mozilla/Firefox_OS/Security"),
+  url_test("/en-US/mobile", "/en-US/docs/Mozilla/Mobile"),
+  url_test("/en-US/mobile/", "/en-US/docs/Mozilla/Mobile"),
+  url_test("/en/mobile/", "/en/docs/Mozilla/Mobile"),
+  url_test("/en-US/addons", "/en-US/Add-ons"),
+  url_test("/en-US/addons/", "/en-US/Add-ons"),
+  url_test("/en/addons/", "/en/Add-ons"),
+  url_test("/en-US/mozilla", "/en-US/docs/Mozilla"),
+  url_test("/en-US/mozilla/", "/en-US/docs/Mozilla"),
+  url_test("/en/mozilla/", "/en/docs/Mozilla"),
+  url_test("/en-US/web", "/en-US/docs/Web"),
+  url_test("/en-US/web/", "/en-US/docs/Web"),
+  url_test("/en/web/", "/en/docs/Web"),
+  url_test("/en-US/learn/html5", "/en-US/docs/Web/Guide/HTML/HTML5"),
+  url_test("/en-US/learn/html5/", "/en-US/docs/Web/Guide/HTML/HTML5"),
+  url_test("/en/learn/html5/", "/en/docs/Web/Guide/HTML/HTML5"),
+  url_test(
+    "/En/JavaScript/Reference/Objects/Array",
+    "/en-US/docs/JavaScript/Reference/Global_Objects/Array"
+  ),
+  url_test(
+    "/En/JavaScript/Reference/Objects",
+    "/en-US/docs/JavaScript/Reference/Global_Objects/Object"
+  ),
+  url_test(
+    "/En/Core_JavaScript_1.5_Reference/Objects/foo",
+    "/en-US/docs/JavaScript/Reference/Global_Objects/foo"
+  ),
+  url_test(
+    "/En/Core_JavaScript_1.5_Reference/foo",
+    "/en-US/docs/JavaScript/Reference/foo"
+  ),
+  url_test("/en-US/HTML5", "/en-US/docs/HTML/HTML5"),
+  url_test("/es/HTML5", "/es/docs/HTML/HTML5"),
+  url_test(
+    "/web-tech/2008/09/12/css-transforms",
+    "/docs/CSS/Using_CSS_transforms"
+  ),
+  url_test("/en-US/docs", "/en-US/docs/Web"),
+  url_test("/es/docs/", "/es/docs/Web"),
+  url_test(
+    "/en-US/devnews/index.php/feed.foo",
+    "https://blog.mozilla.org/feed/"
+  ),
+  url_test("/en-US/devnews/foo", "https://wiki.mozilla.org/Releases"),
+  // Learn is prefixed with /docs which differs from kuma
```

Comment:
would there be a reason to ever redirect to `/en-US/Learn/HTML`?

Reply:
@fiji-flo No, none that I can think of.